packages/create-vite/template-ember-ts/package.json

```diff
@@ -0,0 +1,52 @@
+{
+  "name": "vite-ember-ts-starter",
+  "version": "0.0.0",
+  "private": true,
+  "description": "Small description for app-template goes here",
+  "repository": "",
+  "license": "MIT",
+  "author": "",
+  "type": "module",
+  "imports": {
+    "#src/*": "./src/*",
+    "#config": "./src/config.ts",
+    "#components/*": "./src/components/*"
+  },
+  "exports": {
+    "./tests/*": "./tests/*",
+    "./*": "./src/*"
+  },
+  "scripts": {
+    "start": "vite",
+    "dev": "vite"
```

Comment:
build & preview script missing?

Reply:
I didn't notice they were in other templates.

I've added them now, thank you.